### PR TITLE
Clean doctor-rst whitelist configuration

### PR DESCRIPTION
**Pull request by alamirault**

### CODE REVIEW

File: components/var_dumper.rst [hunk 1]
```diff
@@ -326,11 +326,11 @@ Example::
             // the expected dump contents don't have the default VarDumper structure
             // because of the custom casters and flags used in the test
             $expectedDump = <<<EOTXT
-    [
-      123,
-      "foo",
-    ]
-    EOTXT;
+            [
```
Comment (OskarStark) [inline components/var_dumper.rst]: I thought the same in the past, but ended up with a whitelist entry.

WDYT @javiereguiluz

**Comment by OskarStark** [comment]
Needs a rebase and please revert the change in `var_dumper.rst` for know, so we can merge this PR 👍 

Thanks

**Comment by alamirault** [comment]
I rebased and revert change for `var_dumper.rst`

**Comment by OskarStark** [comment]
Thank you Antoine.

**Pull request merged**